src/tools/gophish_complete.py

```diff
@@ -94,33 +114,41 @@ def select_campaigns(campaigns):
     return inputId
 
 
-def complete_campaign(api, api_key, server, workingID):
-    """Complete a campaign."""
+def complete_campaign(api_key, server, workingID):
```

Comment:
As long as we're cleaning things up in here, can you please change `workingID` to `campaign_id`?

Reply:
Changed variable in commit 6e2412c.

Reply:
I still see `workingID` variable in a few places:  https://github.com/bjb28/gophish-tools/blob/update-gophish-complete/src/tools/gophish_complete.py#L121-L146

Also, missing a period at the end of this line: https://github.com/bjb28/gophish-tools/blob/update-gophish-complete/src/tools/gophish_complete.py#L127

Reply:
Corrected these items in cbc0c60 and used find to double-check this time. :mag:

Reply:
Looks good now!